src/snippets/product-card.liquid

```diff
@@ -1,13 +1,13 @@
 {% unless grid_item_width %}
   {% assign grid_item_width = 'col-sm-6 col-lg-3' %}
+  {% assign vendor = vendor %}
 {% endunless %}
 
 <div class="{{ grid_item_width }}">
   <div class="product-item text-center">
     <a href="{{ product.url | within: collection }}" class="product-item__img relative">
       {% if product.featured_image != blank %}
-        {%
-          render "image",
+        {% render 'image',
```

Comment:
@rdnvol here you need to fix the formatting, this change is not needed.

Reply:
@IhorTretiy 
This formatting is applied after I use the "Format Document" command in the vs code.
This happens because of my plugin, do I need to edit the code in this case?

Reply:
@rdnvol yes, it needs to be changed

Reply:
@rdnvol and what is here?